src/EA.Weee.Web/Areas/Admin/Controllers/AddAatfController.cs

```diff
@@ -37,13 +35,20 @@ public class AddAatfController : AdminController
         private readonly IWeeeCache cache;
         private const int maximumSearchResults = 5;
         private readonly BreadcrumbService breadcrumb;
-
-        public AddAatfController(ISearcher<OrganisationSearchResult> organisationSearcher, Func<IWeeeClient> apiClient, BreadcrumbService breadcrumb, IWeeeCache cache)
+        private readonly IAatfViewModelValidatorWrapper validator;
+
+        public AddAatfController(
+            ISearcher<OrganisationSearchResult> organisationSearcher,
+            Func<IWeeeClient> apiClient,
+            BreadcrumbService breadcrumb,
+            IWeeeCache cache,
+            IAatfViewModelValidatorWrapper validator)
```

Comment:
Do we need need the validator wrapper in the controller if now attached to the view model

Reply:
Removed